Connectors/Rhino/Speckle.Connectors.Rhino7/Bindings/RhinoSelectionBinding.cs

```diff
@@ -30,10 +30,11 @@ ITopLevelExceptionHandler topLevelExceptionHandler
     RhinoDoc.DeselectAllObjects += OnSelectionChange;
   }
 
-  void OnSelectionChange(object o, EventArgs eventArgs)
-  {
-    _idleManager.SubscribeToIdle(() => _topLevelExceptionHandler.CatchUnhandled(UpdateSelection));
-  }
+  void OnSelectionChange(object o, EventArgs eventArgs) =>
+    _idleManager.SubscribeToIdle(
+      nameof(RhinoSelectionBinding),
+      () => _topLevelExceptionHandler.CatchUnhandled(UpdateSelection)
```

Comment:
If we're doing top level exception handling in the idle manager, we could remove the top level here